tests/python_test.py

```diff
@@ -587,3 +587,13 @@ def test_reg_event_parse_args(no_sanlock_daemon, name):
 def test_end_event_parse_args(no_sanlock_daemon, name):
     with raises_sanlock_errno(errno.EALREADY):
         sanlock.end_event(-1, name)
+
+@pytest.mark.parametrize("wait_key", [
+        pytest.param(
+            "async",
+            marks=pytest.mark.xfail(six.PY3,reason="deprecated key")),
+        pytest.param("wait")
+    ])
+def test_rem_lockspace_parse_args(no_sanlock_daemon, wait_key):
+    with raises_sanlock_errno():
+        sanlock.rem_lockspace(b"ls_name", 1, DUMMY_PATH, **{wait_key: False})
```

Comment:
We need to test add_lockspace and remove lockspace with both async and wait on python 2. The best way would be to user parameters for the slow add_rem_lockspace and add_rem_lockspace_async, that will run both options on py2, and only wait= on py3.

Reply:
that would slow tests x 2  for py2

Reply:
It will double the time for one or two tests, not all tests.